binding/nodejs/index.test.js

```diff
@@ -178,6 +200,95 @@ describe("manual paths", () => {
   });
 });
 
+describe("intent detection in DE", () => {
+  test("successful inference beleuchtung", () => {
+    let rhinoEngine = new Rhino(
+      ACCESS_KEY,
+      contextPathBeleuchtungDe,
+      0.5,
+      true,
+      MODEL_PATH_DE
+    );
+
+    let inference = rhinoProcessWaveFile(
+      rhinoEngine,
+      WAV_PATH_IN_CONTEXT_DE
+    );
+
+    // console.log(inference)
+
+    expect(inference["isUnderstood"]).toBe(true);
+    expect(inference["intent"]).toEqual("changeState");
+    expect(inference["slots"]["state"]).toEqual("aus");
+
+    rhinoEngine.release();
+  });
+
+  test("out-of-context phrase is not understood", () => {
+    let rhinoEngine = new Rhino(
+      ACCESS_KEY,
+      contextPathBeleuchtungDe,
+      0.5,
+      true,
+      MODEL_PATH_DE
+    );
+
+    let inference = rhinoProcessWaveFile(
+      rhinoEngine,
+      WAV_PATH_OUT_OF_CONTEXT_DE
+    );
+    expect(inference["isUnderstood"]).toBe(false);
+    expect(inference["intent"]).toBe(undefined);
+
+    rhinoEngine.release();
+  });
+});
+
+describe("intent detection in ES", () => {
+  test("successful inference luz", () => {
+    let rhinoEngine = new Rhino(
+      ACCESS_KEY,
+      contextPathLuzEs,
+      0.5,
+      true,
+      MODEL_PATH_ES
+    );
+
+    let inference = rhinoProcessWaveFile(
+      rhinoEngine,
+      WAV_PATH_IN_CONTEXT_ES
+    );
+
+    console.log(inference)
+
+    expect(inference["isUnderstood"]).toBe(true);
+    expect(inference["intent"]).toEqual("changeColor");
+    expect(inference["slots"]["location"]).toEqual("habitación");
+    expect(inference["slots"]["color"]).toEqual("rosado");
+
+    rhinoEngine.release();
+  });
+
+  test("out-of-context phrase is not understood", () => {
+    let rhinoEngine = new Rhino(
+      ACCESS_KEY,
+      contextPathLuzEs,
+      0.5,
+      true,
+      MODEL_PATH_ES
+    );
+
+    let inference = rhinoProcessWaveFile(
+      rhinoEngine,
+      WAV_PATH_OUT_OF_CONTEXT_ES
+    );
+    expect(inference["isUnderstood"]).toBe(false);
+    expect(inference["intent"]).toBe(undefined);
+
+    rhinoEngine.release();
+  });
+});
+
```

Comment:
No `FR` tests? :cry:

Reply:
FR tests need windows utf-8 fix

Reply:
Ah, makes sense. Strictly speaking, you could change the filename of the french file to use basic ASCII characters instead of the accented ones and it'll work just fine. Do that if you'd like or just make sure there is a ticket in tracking so that we don't forget to do it later.

Reply:
I think it would be good to keep it accented as an extra layer of testing. I'll add it to tracking.